Rakefile

```diff
@@ -0,0 +1,11 @@
+require 'json'
+require 'jasmine'
+
+load 'jasmine/tasks/jasmine.rake'
+
+task :build do
+  puts "Building package"
+  manifest = JSON.parse(File.read("src/manifest.json"))
+  version = manifest['version']
+  sh "cd src && zip -r ../build/govuk-toolkit-chrome-#{version}.zip ."
```

Comment:
Would it not be simpler to do `zip -r build/govuk-toolkit-chrome-#{version}.zip src/*`?